README.Rmd

```diff
@@ -22,7 +22,7 @@ knitr::opts_chunk$set(
 [![CRAN status](https://www.r-pkg.org/badges/version/finalsize)](https://CRAN.R-project.org/package=finalsize)
 <!-- badges: end -->
 
-_finalsize_ provides calculations for the final size of an epidemic in a population with demographic variation in contact patterns, and variation within and between age groups in their susceptibility to disease.
+_finalsize_ provides calculations for the final size of an epidemic in a population, which is the overall number of infected individuals, depending on a demographic distribution (e.g., age groups) and their contact patterns, accounting for different susceptibility to disease between groups (e.g., from different attack rates or infection prevalences) and within groups (e.g., due to immunization programs).
```

Comment:
One question:
- Is the package currently “age group” specific? Because it can also say: "(...) depending on any age-group demographic distribution and their (...)".

Additionally, I'm taking the time to review the sources of different susceptibility between groups. I would like to be sure what is used for setting this parameter. I'll update you on this.

Reply:
No, it's correct to say 'demographic group' I think. So that could be individuals within an age group who have substantially different susceptibility, for example immunosuppressed people, which we might want to represent in the model. A much more complex example of this is in the vignettes for `{fluEvidenceSynthesis}`: https://blackedder.github.io/flu-evidence-synthesis/modelling.html

We've also had Slack discussions to the effect of "this is a simple metapopulation model", so it could be used to represent geographic regions as well. The issue with selling this as a 'spatial-also' model is getting the mixing matrix correct to represent travel flows. I'm happy to include such an example with reliable travel data.

Reply:
my final take on the paragraph edit:

"(...) between groups (e.g., due to age-related impaired or dysregulated immune responses) and (...)" 

instead of 

"(...) between groups (e.g., from different attack rates or infection prevalences) and (...)"

Reply:
Now updated in https://github.com/epiverse-trace/finalsize/pull/121/commits/46cc4529e9b72e0a3a8f0f74e91b8f55ff847c96